dnf5-plugins/automatic_plugin/config/etc/dnf/automatic.conf

```diff
@@ -0,0 +1,104 @@
+[commands]
```

Comment:
May be we should open discussion about location of dnf5 command configurations

Reply:
Sure. I wanted to keep automatic as compatible as possible.

Reply:
`[command]` section is somehow redundant with command-line options.
`random_sleep = 0` has an alias `--timer Apply random delay before execution.`
`download_updates = yes/no` has an alias `--downloadupdates Automatically download updated packages` and `--no-downloadupdates Do not automatically download updated packages`
`apply_updates = yes/no` has an alias  `--installupdates Automatically install downloaded updates` and `--no-installupdates Do not automatically install downloaded updates`

The command `automatic` requires path to configuration file and then it uses overrides over the configuration. I know how it was created, but the question is whether should we keep both path?